assets/css/live_view.scss

```diff
@@ -0,0 +1,87 @@
+/* LiveView specific classes for your customizations */
```

Comment:
Do we need to keep the boilerplate live view styling around, or should I delete it?

Reply:
You probably want to keep or modify the styling for the feedback, phx-click-loading, phx-disconnected, and alert classes.

Reply:
:+1: Thanks!